### ISSUE TITLE
describe.skip or describe.only stop working when using cypress-allure-adapter

### ISSUE THREAD

**Issue by jprealini** [mmisty/cypress-allure-adapter#88]
Hi.. 

I found that when implementing the plugin (0.14.2), Cypress (tried 12.16 and 13.0) starts yelling with 

```
TypeError: The following error originated from your test code, not from Cypress.

  > describe.skip is not a function:
```

I tried by disabling the plugin, and describe.skip works again as expected... 

Any hint?

Thanks

**Comment by mmisty** [comment]
@jprealini Hello!
thank you for reporting! 
I faced that one time but I couldn't find the reproducibility steps.
To find them could you please add more details:
 - test file with which this failed
 - cypress config
 - plugins configuration

Thank you

**Comment by mmisty** [comment]
Ok, I was able to reproduce - this fix is coming

**Comment by mmisty** [comment]
@jprealini this issue should be fixed in 0.16.1

**Comment by jprealini** [comment]
@mmisty well... that was quick! :D Awesome.. I just tested it and it's working as expected. Thanks!!
